blocks.md

```diff
@@ -384,40 +394,35 @@ Creates a block which displays disk space information.
 [[block]]
 block = "disk_space"
 path = "/"
-alias = "/"
 info_type = "used"
 unit = "GiB"
-format = "{icon}{used}/{total} {unit} ({available}{unit} free)"
+format = "{icon} {used}/{total} ({available} free)"
 ```
 
 #### Options
 
 Key | Values | Required | Default
 ----|--------|----------|--------
 `alert` | Available disk space critical level as a percentage or Unit. | No | `10.0`
-`alias` | Alias that is displayed for path. | No | `"/"`
```

Comment:
I thought we were keeping this around for now and deprecating in a later release?

Reply:
Ah, yeah, I'll move it back.

Reply:
Can create a `#### Deprecated Format Keys` section for it, like those other blocks

Reply:
By the way, when deprecation placeholders, it is OK to change the default format not to include it?

Reply:
So there is no way to have it display the same thing using different placeholders? What's the before/after look like in this case?

Reply:
> So there is no way to have it display the same thing using different placeholders?

There is, just asking.

Reply:
In that case the best thing is to have the output be the same :)